app/assets/javascripts/polaris_view_components/autocomplete_controller.js

```diff
@@ -60,12 +60,31 @@ export default class extends Controller {
     return this.application.getControllerForElementAndIdentifier(this.popoverTarget, 'polaris-popover')
   }
 
+  get resultsTarget() {
+    return this.popoverController.popoverTarget.querySelector('[data-polaris-autocomplete-target="results"]')
```

Comment:
Maybe let's replace `data-polaris-autocomplete-target` with `data-target`? This way it will be clear that this is not Stimulus but custom target.

Reply:
That might be better yeah 😄